Makefile

```diff
@@ -41,6 +41,10 @@ DOCKER_CONFIG ?= "$(HOME)/.docker"
 
 # Mention the vm-driver that should be used to install OpenShift
 vm-driver ?= "kvm"
+
+COMMIT_ID ?= "317cc36"
```

Comment:
why hardcoded? IMO the default value should be the latest commit at that time.

Reply:
Is there a way we can find the commit ID from a release version? We shouldn't have the latest because there's no guarantee it will be stable.

Reply:
We can get commit ID from a release version but for the case now and few other cases as well, we don't necessarily want to pick the commit from released tag, so left it open to use any commit we want to pick.